.github/workflows/main.yml

```diff
@@ -9,17 +9,30 @@ on:
       - master
 
 jobs:
+  ruby-versions:
+    uses: ruby/actions/.github/workflows/ruby_versions.yml@master
+    with:
+      min_version: 2.7
```

Comment:
I think we can specify `engine` here to `cruby`, then we won't need to exclude truffleruby below.

Reply:
Thanks, yeah, good idea!